src/components/Image/Image.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import classnames from 'classnames';
+
+export interface ImageProps {
+	src: string;
+	alt?: string;
+	wide?: boolean;
+}
+
+export const Image: FunctionComponent<ImageProps> = ({ src, alt, wide }: ImageProps) => (
+	<div className={classnames({ 'c-image': !wide, 'c-image--full-width': wide })}>
```

Comment:
```
classnames('c-image', { 'c-image--full-width': wide })
```
Dit is logischer en heeft hetzelfde eindresultaat op de pagina.
Hun styleguide is gewoon niet erg logisch opgebouwd. 
Zouden we dan niet de meer logische oplossing gebruiken?
Dan kunnen we alle images identificeren met een enkele class: .c-image

Reply:
Ik ging dat eerst ook zo doen, maar dan zag ik dat die `c-image` ook nog de margins aanpast. Bij mij zat er wel degelijk een klein verschil op (door de 0 op vertical margins). Ik vind het ook logischer om die base-class te laten staan (zoals het zou moeten zijn), dus ik voeg hem toe!